cmd/collector/app/span_processor.go

```diff
@@ -250,11 +250,37 @@ func (sp *spanProcessor) ProcessSpans(ctx context.Context, batch processor.Batch
 		// TODO verify if the context will survive all the way to the consumer threads.
 		ctx := tenancy.WithTenant(ctx, batch.GetTenant())
 
+		// BUGFIX: Collect metrics for V2 spans that were being skipped
```

Comment:
this is unrelated to your PR, please use git merges correctly to only reflect your changes